<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rsc.qrc">
    <normaloff>:/icon/Images/wind_icon.png</normaloff>:/icon/Images/wind_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(11, 11, 11);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="top_bar">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(11, 11, 11);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_erro">
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 84, 84);
border-radius: 10px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_erro">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Erro</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_cl_popup">
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">image: url(:/close/Images/close-icon.png);
QPushButton {
	border-radius: 5px;
	background-image: url(:/close-icon/Images/close-icon.png);
	background-position: center;
	background-color: rgb(190, 0, 0);
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(165, 0, 0);
}
QPushButton:pressed {
	color: rgb(255, 255, 255);
	background-color: rgb(190, 0, 0);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="content">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="login_area">
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>500</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(83, 83, 83);
border-radius: 10px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QFrame" name="frame_icon">
          <property name="geometry">
           <rect>
            <x>155</x>
            <y>35</y>
            <width>90</width>
            <height>100</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">image: url(:/user/Images/user-icon.png);
background-repeat: no-repeat;
background-position: center;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
         <widget class="QLineEdit" name="line_user">
          <property name="geometry">
           <rect>
            <x>85</x>
            <y>200</y>
            <width>230</width>
            <height>40</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	border: 2px solid rgb(45, 45, 45);
	border-radius: 5px;
	background-color: rgb(30, 30, 30);
	color: rgb(200, 200, 200);
	padding: 8px;
}
QLineEdit:hover {
	border: 2px solid rgb(200, 200, 200);
	color: rgb(200, 200, 200);
}
QLineEdit:focus {
	border: 2px solid rgb(254, 14, 15);
	color: rgb(200, 200, 200);
}
</string>
          </property>
          <property name="maxLength">
           <number>12</number>
          </property>
          <property name="placeholderText">
           <string>USER:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="line_pass">
          <property name="geometry">
           <rect>
            <x>85</x>
            <y>250</y>
            <width>230</width>
            <height>40</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	border: 2px solid rgb(45, 45, 45);
	border-radius: 5px;
	background-color: rgb(30, 30, 30);
	color: rgb(200, 200, 200);
	padding: 8px;
}
QLineEdit:hover {
	border: 2px solid rgb(200, 200, 200);
	color: rgb(200, 200, 200);
}
QLineEdit:focus {
	border: 2px solid rgb(254, 14, 15);
	color: rgb(200, 200, 200);
}
</string>
          </property>
          <property name="maxLength">
           <number>12</number>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>PASS:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>150</y>
            <width>200</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Calibri Light</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>CRUD</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QPushButton" name="bt_enter_login">
          <property name="geometry">
           <rect>
            <x>85</x>
            <y>320</y>
            <width>230</width>
            <height>40</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	border: 2px solid rgb(45, 45, 45);
	border-radius: 5px;
	background-color: rgb(254, 14, 15);
	color: rgb(30, 30, 30);
	padding: 8px;
}
QPushButton:hover {
	border: 2px solid rgb(200, 200, 200);
	color: rgb(200, 200, 200);
}
QPushButton:pressed {
	border: 2px solid rgb(45, 45, 45);
	color: rgb(30, 30, 30);
}</string>
          </property>
          <property name="text">
           <string>ENTER</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="bottom">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(21, 21, 21);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_credits">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>© 2023 Created by: Gualter Ronan.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../rsc.qrc"/>
 </resources>
 <connections/>
</ui>
